aggregations/db_tables/daily_transaction_fees.py

```diff
@@ -0,0 +1,45 @@
+from . import DAY_LEN_SECONDS, daily_start_of_range
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyTransactionFees(PeriodicAggregations):
+    @property
+    def sql_create_table(self):
+        # Use numeric(50,0) per discussion in https://github.com/near/near-analytics/issues/17.
+        return """
+            CREATE TABLE IF NOT EXISTS daily_transaction_fees
+            (
+                collected_for_day   DATE PRIMARY KEY,
+                transaction_fees    numeric(50, 0) NOT NULL
```

Comment:
Let's think a little bit more about naming.
1. Now it's unclear, what exactly do we store here. It's tokens, so we should definitely mention `tokens` here.
2. `tokens_burnt` is not OK since not all of them are really burnt, part of this amount will go to royalty https://github.com/telezhnaya/docs/blob/master/docs/tokens/balances.md#calling-a-function.
3. We also want to mention `fee` here, that's our goal to compute.
3. Are they always about transactions? It should be true for now, but I think it's better not to stick to it.

`tokens_spent_on_fees` ? And the table could be `daily_tokens_spent_on_fees`.

@rbval I want to know your opinion!
@frol guru of naming, we need your help here 🙏

Reply:
> It's a combination of gas_used and gas_price. Is it guaranteed that it's always tokens?

Yes, the gas price is always in NEAR tokens per gas unit, so [gas-unit * NEAR/gas-unit] = NEAR.

I agree that we need to indicate the nature of the number we store, and indeed, we use `tokens` to indicate that the value is in NEAR tokens, and more precisely, in yoctoNEAR to avoid any rounding issues.

I like the proposed `tokens_spent_on_fees`, though we might also consider `tokens_spent_on_transaction_fees`, but I don't have a strong opinion on this variation. I definitely see how the original `transaction_fees` can be ambiguous, but `tokens_spent_on_fees` is specific enough, so I will live it up to you.

Reply:
I think `tokens_spent_on_fees` makes most sense in this case. PR updated!